app/components/schools/mentors/ect_mentor_training_details_component.rb

```diff
@@ -15,7 +15,7 @@ def render?
       end
 
       def eligible_for_training?
-        teacher.mentor_completion_date.nil?
+        teacher.mentor_became_ineligible_for_funding_on.nil?
       end
```

Comment:
Should this be checking that the date is earlier than today?

Reply:
Sorry just noticed this is still in draft - if you'd already spotted it ignore me 🫥

Reply:
Good question, it's one I also asked: 

See my post on slack here: https://ukgovernmentdfe.slack.com/archives/C06V31R0Z17/p1744210652433659?thread_ts=1744208328.640409&cid=C06V31R0Z17

Do you think it should be changed?

Reply:
Resolving this. As it was discussed on slack.

We will create a new card/cards to make things clearer here as part of future work.